### PR TITLE
chore: prevent warning on editor mode

### PR DESCRIPTION
**Pull request by eordano**
Signed-off-by: Esteban Ordano <esteban@decentraland.org>

### CODE REVIEW

**Review by popuz — APPROVED** [review]
LGTM 🟢

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs [hunk 1]
```diff
@@ -56,6 +59,11 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
 
     public void Initialize()
     {
+        #if UNITY_EDITOR
```
Comment (mikhail-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs:62]: Why it is for Unity Editor only? What about builds?

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs [hunk 2]
```diff
@@ -56,6 +59,11 @@ public class ExploreV2MenuComponentController : IExploreV2MenuComponentControlle
...
+        // Prevent warning when starting on unity editor mode
+        // TODO: Are we instantiating 500 different kinds of animations?
+        DOTween.SetTweensCapacity(500,50);
```
Comment (mikhail-dcl) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs:65]: Please move it to `Main`, it has nothing to do in `ExploreV2MenuComponentController`

**Review by mikhail-dcl — CHANGES_REQUESTED** [review]

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs (hunk 1 above)
Comment (eordano) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs]: This change is in the scope of https://github.com/decentraland/unity-renderer/issues/4304 -- I left the TODO in there to analyze if it should be done also for the build, as 500 instances created looked a little too much IMHO but I'm not familiar with how are they getting used

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs (hunk 2 above)
Comment (eordano) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/MainMenu/ExploreV2Menu/ExploreV2MenuComponentController.cs]: Fixed!

**Review by mikhail-dcl — APPROVED** [review]

**Pull request merged**